### PR TITLE
Updating Lenovo Quick Clean Manifests

### PR DESCRIPTION
**Pull request by philjorgensen**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95074)

### CODE REVIEW

**Comment by philjorgensen** [comment]
@stephengillie I tested this using the SandboxTest.ps1 and everything installed fine. What am I missing here?

File: manifests/l/Lenovo/QuickClean/1.02.0051/Lenovo.QuickClean.installer.yaml [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-  Silent: /SP- /VERYSILENT /NORESTART
-  SilentWithProgress: /SP- /SILENT /NORESTART
-  InstallLocation: C:\Program Files (x86)\Lenovo\Lenovo Quick Clean
+  Silent: /VERYSILENT /NORESTART
+  SilentWithProgress: /SILENT /NORESTART
+  InstallLocation: /DIR="<INSTALLPATH>"
+  Log: /LOG="<LOGPATH>"
 Installers:
-- Architecture: x64
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/l/Lenovo/QuickClean/1.02.0051/Lenovo.QuickClean.installer.yaml:17]:
```suggestion
- Architecture: x64
```

Comment (stephengillie) [reply manifests/l/Lenovo/QuickClean/1.02.0051/Lenovo.QuickClean.installer.yaml:17]: Packages with x86 architecture are validated on a 32-bit OS. This seems to be a 64-bit package:
image

Comment (philjorgensen) [reply manifests/l/Lenovo/QuickClean/1.02.0051/Lenovo.QuickClean.installer.yaml:17]: Ah, ok. Wingetcreate identifies it as x86. This has happened with other packages of ours. Thanks for helping!

**Review by Trenly — APPROVED** [review]

**Pull request merged**